### ISSUE TITLE
Volume mount icon not visible anymore since update to osxfuse 3.10.6

### ISSUE THREAD

**Issue by mixtly87** [osxfuse/osxfuse#701]
Since I updated osxfuse to 3.10.6, my app's volume mount icon is not showing anymore. Instead, the default one is shown.

Here is the mount-volume code:
```
NSString* volName = [NSString stringWithFormat:@"volname=%@", mountPath.lastPathComponent];
NSString* mountIconPath = [[NSBundle mainBundle] pathForResource:@"mount-icon" ofType:@"icns"];
NSString* volIcon = [NSString stringWithFormat:@"volicon=%@", mountIconPath];
NSArray* options = @[ volName, @"modules=iconv", volIcon, @"auto_cache", @"daemon_timeout=30" ];
[self.fuse mountAtPath:mountPath withOptions:options];
```

Is anyone else experiencing similar issue?

**Comment by mcicak** [comment]
I'm also experiencing the same issue with osxfuse filesystems examples.

**Comment by kiril-kirov** [comment]
Same here, I can confirm the missing icon as well.

**Comment by jarrocha** [comment]
Try adding the **local** mount option

**Comment by bfleischer** [comment]
This issue was introduced in version 3.10.5 and is fixed in version 3.11.0.
